src/config.template.yaml

```diff
@@ -64,6 +64,23 @@ platforms:
       # you can use {resources_path} for the path to the AlexaPi "resources" directory
       audio_file: ""
 
+    # Commands to run before and after an interaction begins
+    interaction_commands:
+
+      # Example used to lower shairport-sync volume during interaction with alexa
+      #     pre_command: "amixer set shairportvolume 10%-""
+      #     post_command: "amixer set shairportvolume 10%+""
+      # Note this assumes you have redirected shairport-syncs audio to a seperate softdevice
+      # with control "shairportvolume". Example asound.conf avalible in the scripts folder
+
+      # Command to run after a trigger is detected but before an interaction begins
+      pre_enabled: false
```

Comment:
Lose the `_enabled` variables. If people want to disable it, they can just comment it out. If the `_command` is there and non-empty, we run it.